test/functional/rpc_net.py

```diff
@@ -62,8 +62,11 @@ def _test_getnettotals(self):
         # the bytes sent/received should change
         # note ping and pong are 32 bytes each
         self.nodes[0].ping()
-        wait_until(lambda: (self.nodes[0].getnettotals()['totalbytessent'] >= net_totals_after['totalbytessent'] + 32 * 2), timeout=1)
-        wait_until(lambda: (self.nodes[0].getnettotals()['totalbytesrecv'] >= net_totals_after['totalbytesrecv'] + 32 * 2), timeout=1)
+        wait_until(lambda: (
+            self.nodes[0].getnettotals()['totalbytessent'] >= net_totals_after['totalbytessent'] + 32 * 2
+            and self.nodes[0].getnettotals()['totalbytesrecv'] >= net_totals_after['totalbytesrecv'] + 32 * 2
+            and ('pong' in self.nodes[0].getpeerinfo()[0]['bytesrecv_per_msg'])
```

Comment:
Instead of `and`, could split them into separate lines with separate `wait_until`s to see what condition actually fails (if one does)?

Reply:
Ah, right, didn't consider that. Fixed.